conda/activate.py

```diff
@@ -890,10 +911,13 @@ class PosixActivator(_Activator):
     tempfile_extension = None  # output to stdout
     command_join = "\n"
 
-    unset_var_tmpl = "unset %s"
+    # Using `unset %s` would cause issues for people running
+    # with shell flag -u set (error on unset).
+    unset_var_tmpl = "export %s=''"  # unset %s
     export_var_tmpl = "export %s='%s'"
+    path_var_tmpl = "export %s=\"$(cygpath '%s')\"" if on_win else export_var_tmpl
     set_var_tmpl = "%s='%s'"
-    run_script_tmpl = '. "%s"'
+    run_script_tmpl = ". \"`cygpath '%s'`\"" if on_win else '. "%s"'
```

Comment:
Pretty sure this line is causing the test errors. Why was`cygpath` added here?

Reply:
Not this line, but the one further down in the `CshActivator` on line 962

You're right the code didn't previously do this. This was added to mirror what was already being done for `export`.